api/v1/ptpoperatorconfig_types.go

```diff
@@ -29,6 +29,8 @@ type PtpOperatorConfigSpec struct {
 	// Important: Run "make" to regenerate code after modifying this file
 
 	DaemonNodeSelector map[string]string `json:"daemonNodeSelector"`
+	// +kubebuilder:default={}
```

Comment:
I think you can remove this it can be just nil

Reply:
Fixed

Reply:
fixed

Reply:
I can still see it :P

Reply:
fixed